Override postcss to ^8.4.31.

@generated
@generator: scripts/apply-override.sh --patch common/override-postcss.diff --override 'global:postcss=^8.4.31'
@override-package: postcss@^8.4.31

Index: code-editor-src/package.json
===================================================================
--- code-editor-src.orig/package.json
+++ code-editor-src/package.json
@@ -237,6 +237,7 @@
   },
   "overrides": {
     "braces": "^3.0.3",
+    "postcss": "^8.4.31",
     "node-gyp-build": "4.8.1",
     "serialize-javascript": "^7.0.3"
   },
